src/applications/gi/sass/gi.scss

```diff
@@ -271,3 +271,12 @@
     cursor: pointer;
   }
 }
+
+.lc-wrapper {
```

Comment:
might be able to use `vads-u-display--flex vads-u-flex-direction--column` instead

Reply:
You're right. The change has been made